tests/admin/test_controller.py

```diff
@@ -2124,14 +2361,17 @@ def setup(self):
         self.responses = []
         self.requests = []
 
+        # Make the admin a system admin so they can do everything by default.
+        self.admin.add_role(AdminRole.SYSTEM_ADMIN)
+
     def do_request(self, url, *args, **kwargs):
         """Mock HTTP get/post method to replace HTTP.get_with_timeout or post_with_timeout."""
         self.requests.append(url)
         response = self.responses.pop()
         return HTTP.process_debuggable_response(response)
 
     def test_get_integration_protocols(self):
-        """Test the _get_integration_protocols helper method."""
+        # Test the _get_integration_protocols helper method.
```

Comment:
This annoys me as well, but a docstring just seems better from a maintanability perspective. I'm not gonna nitpick in the context of this review but we should be able to figure out a different way to run the tests or something to get the display we want.

Reply:
The actual reason I did this was that it made it easier to comment out large chunks of the file while debugging, and I decided to leave it because I liked it better anyway. I'm fine with changing it back for now since it's unrelated to this pr.